docs/platform/quickstart.mdx

```diff
@@ -230,6 +230,101 @@ curl -X POST "https://api.mem0.ai/v1/memories/search/" \
 ```
 </CodeGroup>
 
+#### Search using custom filters
+
+You can also search for memories using custom filters along with user_id, agent_id, app_id, etc. 
+
+Here you need to define `version` as `v2` in the search method.
+
+<CodeGroup>
+
+```python Python
+query = "What do you know about me?"
+filters = {
+   "AND":[
+      {
+         "user_id":"alex"
+      },
+      {
+         "agent_id":{
+            "in":[
```

Comment:
Can we add more examples that involves other kinds of filters such as `gt`, `gte`, `eq` etc etc.